cmd/rook/ceph/osd_test.go

```diff
@@ -49,8 +49,8 @@ func TestParseDesiredDevices(t *testing.T) {
 	assert.Nil(t, result)
 	assert.NotNil(t, err)
 
-	// metadataDevice and OSDsPerDevice
-	devices = "sdd:1:sdb,sde:1:sdb,sdf:1:sdc,sdg:1:sdc"
+	// metadataDevice, databaseSizeMB, deviceClass and OSDsPerDevice
```

Comment:
the order is this, right? osdsPerDevice, databaseSizeMB, deviceClass, metadataDevice

Reply:
Yes. I will fix